lib/venue.rb

```diff
@@ -1,2 +1,25 @@
 class Venue
-end
+  attr_reader :name, :capacity, :patrons
+  def initialize(name, capacity, patrons = [])
+    @name = name
+    @capacity = capacity
+    @patrons = patrons
+  end
+  def add_patron(patron)
+    @patrons += [patron]
+  end
+  def yell_at_patrons
+    @patrons.each do |patron|
+      patron.upcase
+      @patrons << patron
+    end
+  end
+  def over_capacity?
+    if @patrons >= 4
```

Comment:
Nice start to this. The way you have this written you are doing what is called `hard coding` the value. Is there some variable that already exists that stores the value of 4? Hint: what is `@capacity`?